test/rx/util/health_check_authorization_test.rb

```diff
@@ -0,0 +1,21 @@
+require "test_helper"
+
+class HealthCheckAuthorizationTest < Minitest::Test
+  def setup
+    @env = { "HTTP_AUTHORIZATION" => "123" }
+  end
+
+  def test_it_checks_the_default_authorization
+    @check = Rx::Util::HealthCheckAuthorization.new(@env, "123")

```

Comment:
this test is failing because the class checks the `HTTP_AUTHORIZATION_TOKEN` key instead of the `HTTP_AUTHORIZATION` token you're using here.

Reply:
Yeah, when I wrote these tests I was using the `HTTP_AUTHORIZATION` header and I changed it and forgot to adapt the tests. fixed now